other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -50,8 +50,27 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        print("Please select a film material. Film is set to substrate.")
</pre><p>Film material not found. Re-using substrate material as film.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>ok</p>